tests/offline/test_cli.py

```diff
@@ -112,6 +113,11 @@ def test_filestatus(m: Maestral) -> None:
 
 
 def test_autostart(m: Maestral) -> None:
```

Comment:
The canonical way to skip tests with pytest is a `@pytest.mark.skipif()` decorator. When using, tests will clearly show as skipped in the test output instead just always passing.

Reply:
Aha! Good to know.  Not a Python developer :-) will tweak.

Reply:
Yeah, pytest is pretty neat.